modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -57,6 +57,21 @@ function is_writable_mongo {
   mongo --quiet --eval "print(db.isMaster()[\"ismaster\"]);" "localhost/$database"
 }
 
+function is_writable_elasticsearch {
```

Comment:
Also, this function is using the hostname from the AWS meta data to decide whether the node is the master. This might cause a problem if node 1 is not the master.

Reply:
There is no master/slave, we arbitrary choose one node (number 1) just to avoid restoring on several nodes at the same time